### PR TITLE
BUG FIX: Correct spacing next to symbols when styles switch.

### PR DESCRIPTION
**Pull request by kohler**
Two nodes were sharing a list of classes, which led to the style
change applying multiple times. Fix that.

(This is my bug, from the original spacing PR.)

### CODE REVIEW

**Comment by kohler** [comment]
Cc @kevinbarabash

File: src/buildHTML.js [hunk 1]
```diff
@@ -83,7 +83,7 @@ var buildExpression = function(expression, options, isRealGroup) {
             i--;
         } else if (spaces) {
             if (groups[i] instanceof domTree.symbolNode) {
-                groups[i] = makeSpan(groups[i].classes, [groups[i]]);
+                groups[i] = makeSpan([].concat(groups[i].classes), [groups[i]]);
```
Comment (kevinbarabash) [inline src/buildHTML.js]: How was `groups[i].classes` getting mutated?

Comment (kohler) [inline src/buildHTML.js]: Should this be written as `groups[i].classes.slice()` instead?

**Comment by kevinbarabash** [comment]
@kohler thanks for noticing the issue and fixing it so promptly.

**Pull request merged**